packages/v4/patternfly-docs/content/accessibility/card/card.md

```diff
@@ -3,21 +3,109 @@ id: Card
+| `component` | `Card` | Sets the base element for the sub-component (`article` element by default). If you primarily want to use a card for styling or layout purposes, it's recommended to pass in a value of "div" to this prop. |
+| `hasSelectableInput` | `Card` | Adds a visually hidden input that notifies users navigating via assistive technologies that a card is selectable. This prop should only be passed in when only one card within a set of cards should be selectable at a time, such as for a [primary-detail view](/demos/primary-detail). <br/><br/> When this prop is passed in, either a `CardTitle` sub-component must exist in the card, or the `selectableInputAriaLabel` prop must be passed in. The hidden input will automatically be given an accessible name when a `CardTitle` exists within the card, otherwise the value given to the `selectableInputAriaLabel` prop will add an accessible name to the input. |
+| `isDisabledRaised` | `Card` | Adds styling to visually indicate that a selectable card is disabled. When the `hasSelectableInput` prop is passed in, this prop will also set the selectable input's `disabled` attribute. |
+| `isExpanded` | `Card` | Adds styling to a card toggle to visually indicate whether the expandable content is expanded or collapsed. When this prop is passed in, `onExpand` and `toggleButtonsProps` must be passed into the `CardHeader` sub-component. |
+| `isSelectable` or `isSelectableRaised` | `Card` | Adds styling on hover or focus to visually indicate that a card is selectable, and adds the `tabindex="0"` attribute so that the card can be navigated to with standard keyboard navigation. |
+| `isSelected` | `Card` | Adds styling to visually indicate that a card is currently selected. When the `hasSelectableInput` prop is passed in, this prop will also set the selectable input's `checked` attribute. |
+| `selectableInputAriaLabel` | `Card` | Adds an accessible name to the hidden input when `hasSelectableInput` is passed in and there is no `CardTitle` sub-component within the card. The value passed into this prop should act as a title for the card. |
+| `component` | `CardBody` | Sets the base element for the sub-component (`div` element by default). |
+| `component` | `CardFooter` | Sets the base element for the sub-component (`div` element by default). |
+| `onExpand` | `CardHeader` | Adds a toggle button with a caret icon to control whether a card is expanded or collapsed, and to visually indicate the card is expandable. <br/><br/> If an expandable card does not include a `CardTitle`, there should be some other visual context in the `CardHeader` to convey the content of the card to users that rely on sight, such as an image. |
+| `toggleButtonProps` | `CardHeader` | Sets props on the card's toggle. **Required** when a card is expandable. |
```

Comment:
It's not clear from this description when/how the `toggleButtonProps` prop has any affect on a11y.

Reply:
I see now that you have more explanation about that below. Maybe you should note that a11y considerations for toggleButtonProps can be found below - possibly even link to that section header?